app/frontend/packs/nationality-autocomplete.js

```diff
@@ -6,7 +6,8 @@ const initNationalityAutocomplete = () => {
       "#candidate-interface-personal-details-form-first-nationality-field",
       "#candidate-interface-personal-details-form-first-nationality-field-error",
       "#candidate-interface-personal-details-form-second-nationality-field",
-      "#candidate-interface-personal-details-form-second-nationality-field-error"
+      "#candidate-interface-personal-details-form-second-nationality-field-error",
+      "#candidate-interface-contact-details-form-country-field"
```

Comment:
Is there an -error version of this new field?

Reply:
Yes there is, thanks for spotting that